packages/_server/src/config/cspellConfig.ts

```diff
@@ -691,9 +710,9 @@ interface CSpellSettingsPackageProperties extends CSpellSettings {
  */
 type GlobDefX = GlobDef;
 
-export interface CustomDictionaryWithScope extends CustomDictionary {}
+export interface CustomDictionaryWithScope extends CustomDictionary { }
```

Comment:
If you run `yarn lint` is should fix any issues.

Reply:
Will do it this time. A suggestion would be using git hook, or, add it to the "postinstall".